L/LRSLib_PolyhedraFork/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LRS_PolyhedraFork"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/JuliaPolyhedra/lrslib.git", "d8b723a2c315614979a8354f9e768d273d14a215"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd lrslib
+install_license COPYING
+
+if [ $(uname) = "Darwin" ]; then
+    atomic_patch -p1 ../makefile.osx.patch
+fi
+
+make liblrs.${dlext}.0 SONAME=liblrs.${dlext} SHLIB=liblrs.${dlext}.0
+cp liblrs.${dlext}.0 ${libdir}/liblrs.${dlext}
+
+cp ../makefile.liblrsnash .
+make -f makefile.liblrsnash SHLIB=liblrsnash.${dlext} LIBLRSDIR=${libdir}
+cp liblrsnash.${dlext} ${libdir}/liblrsnash.${dlext}
```

Comment:
Compilation is just a single command, isn't it easier to just run it directly here instead of moving makefiles around just to run that single command?